src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -208,37 +212,39 @@ def _create_users_tasks(self) -> List[asyncio.Task[None]]:
             )
         return tasks
 
-    def _get_upper_bound(self, batteries: Set[int]) -> float:
+    def _get_upper_bound(self, batteries: Set[int], use_all: bool) -> float:
         """Get total upper bound of power to be set for given batteries.
 
         Note, output of that function doesn't guarantee that this bound will be
         the same when the request is processed.
 
         Args:
             batteries: List of batteries
+            use_all: flag whether all batteries must be used for the power request.
```

Comment:
the documentation should be: `use_all: whether all batteries in the power request must be used.`

This comment applies for all places where `use_all` has been set